partcad-cli/src/partcad_cli/click/commands/inspect.py

```
@@ -0,0 +1,109 @@
+def cli(ctx, context, verbal, package, interface, assembly, sketch, params, object):
+    params = {}
+    if not params is None:
+        for kv in params:
+            k, v = kv.split("=")
+            params[k] = v
+
+    if package is None:
+        if ":" in object:
+            path = object
+        else:
+            path = ":" + object
```

Ruff (0.8.0)

Use ternary operator `path = object if ":" in object else ":" + object` instead of `if`-`else`-block

Replace `if`-`else`-block with `path = object if ":" in object else ":" + object`

(SIM108)